### DISCUSSION TITLE
MAUI Crash inRelease Mode

### DISCUSSION THREAD

**Discussion by MohMehrnia** [dotnet/maui#9719] [General]
My MAUI app crashed on release mode on android, but in debug mode is OK, why?
WHEN change Release to apk splash screen is freeze and when bundle app crashed, but in debug mode every thins is OK

**Comment by rpascalegh** [comment]
I am having same issue.  App run correctly in Debug.  When I go to release the splash screen comes on and android app crashes.  Anyone have fix?

**Comment by Mafyou** [comment]
Same here.

**Comment by davidortinau** [comment]
This is often (not always) an AOT issue where something is being trimmed out for release mode. Look at your crash dump and identify which dependency might be missing, or other root cause.

**Comment by Mafyou** [comment]
I found: just delete bin/obj.

**Comment by woorim02** [comment]
When there is an error in XAML, errors can occur in release builds. 
In my case, I used a Style Key in Style="{StaticResource ...}" that does not exist in ContentPage.Resources.
While this ran in debug builds, it crashed on release mode
